features/step_definitions/git-flow.js

```diff
@@ -200,4 +186,22 @@ module.exports = function server() {
             Assert.oneOf(data.state, ['success', 'pending']);
         })
     );
+
+    // eslint-disable-next-line new-cap
+    this.After({
```

Comment:
Cannot figure out a better way to do it. :(  We don't know the pipelineId before we create it in the `.Before` hook